src/librustc_ast/ast.rs

```diff
@@ -313,15 +313,15 @@ pub type GenericBounds = Vec<GenericBound>;
 pub enum ParamKindOrd {
     Lifetime,
     Type,
-    Const,
+    Const { unordered: bool },
```

Comment:
can you also add a comment here mentioning that `min_const_generics` requires const params to come after types.